libs/logging/src/logger.ts

```diff
@@ -1,6 +1,6 @@
 import makeDebug from 'debug';
 import { Dictionary } from '@votingworks/types';
-import { getDetailsForEventId, LogLine } from '.';
+import { CLIENT_SIDE_LOG_SOURCES, getDetailsForEventId, LogLine } from '.';
```

Comment:
Import from the files that contain these rather than the re-exporting `index.ts`?